lib/matplotlib/backends/qt_compat.py

```diff
@@ -180,6 +181,15 @@ def is_pyqt5():
     raise AssertionError("Unexpected QT_API: {}".format(QT_API))
 
 
+# Fixes issues with Big Sur
+# https://bugreports.qt.io/browse/QTBUG-87014, fixed in qt 5.15.2
+if (sys.platform == 'darwin' and
+        LooseVersion(platform.mac_ver()[0]) >= LooseVersion("10.16") and
```

Comment:
I don't know if this should be a `>=` or `==` and I don't know if we can even know how is it going with to be next MacOS release...